scripts/executeService.js

```diff
@@ -0,0 +1,22 @@
+const IdentityProxy = require('../build/IdentityProxy.json');
+
+const ethers = require('ethers');
+const providers = ethers.providers;
+const Wallet = ethers.Wallet;
+const utils = ethers.utils;
+const keys = require('./../restApi/config/keys.js')
+
+let serviceAddress = keys.serviceAddress;
+
+var provider;
+
+async function executeService(identityProxyAddress, serviceContractAddress, reward, wei, data, signedDataHash, deployerWallet){
+	provider = new providers.JsonRpcProvider('http://localhost:8545', providers.networks.unspecified);
```

Comment:
The url of the provider should be passed by config and default to `http://localhost:8545`